recipes/argh/all/conanfile.py

```diff
@@ -0,0 +1,36 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class ArgparseConan(ConanFile):
+    name = "argh"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/adishavit/argh"
+    topics = ("conan", "argh", "argument", "parsing")
+    license = "BSD-3"
+    description = "Frustration-free command line processing"
+    settings = "compiler"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return os.path.join(self.source_folder, "source_subfolder")
+
+    def configure(self):
```

Comment:
```suggestion
    def validate(self):
```
https://blog.conan.io/2020/12/14/New-conan-release-1-32.html